extensions/dbgobject/type/dbgobject-type.js

```diff
@@ -26,7 +26,8 @@ Loader.OnLoad(function () {
 
         // Normalize type name.
         this._name = name.trim();  // strip outer whitespace
-        this._name = this._name.replace(/^const\s+/, "");  // remove leading "const" and whitespace
+        this._name = this._name.replace(/&/g, "*"); // treat references like pointers
+        this._name = this._name.replace(/^((const|volatile)\s+)+/g, "");  // remove leading "const" and whitespace
```

Comment:
I think technically "const volatile" is possible too. :)

Reply:
Yes, the last + in the regex and the /g handle that case.

Reply:
Missed that, sorry. That looks good.